app/controllers/converts_controller.rb

```diff
@@ -28,6 +28,9 @@ def create
       filename: file_name,
       layout: false
     )
+  rescue ActionController::ParameterMissing
+    flash.now[:alert] = "Missing 'file' or 'format' form option."
```

Comment:
Did you mean the error message should be nicer in terms of language or more specific?